CHANGELOG.md

```diff
@@ -8,12 +8,8 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [3.0.0]
```

Comment:
This can be a minor version bump since we're no longer introducing breaking changes.
```suggestion
## [2.1.0]
```